src/nft/css/sprinkles.css.ts

```diff
@@ -401,6 +403,7 @@ const unresponsiveProperties = defineProperties({
     borderBottom: vars.border,
     borderTop: vars.border,
     borderWidth,
+    borderLeftWidth: borderWidth,
```

Comment:
Small nit: looks like borderLeftWidth is added, but not used anywhere. Otherwise LGTM